workflow/build.gradle

```diff
@@ -12,8 +12,9 @@ dependencies {
     implementation 'org.springframework:spring-tx'
     implementation 'jakarta.annotation:jakarta.annotation-api'
     implementation 'jakarta.jms:jakarta.jms-api'
-    implementation 'com.synopsys.integration:integration-rest'
-    implementation 'com.synopsys.integration:integration-common'
+
+    implementation 'com.synopsys.integration:blackduck-common', rootProject.ext.blackduckCommonExcludes
```

Comment:
Would it be possible to make this `testImplementation`?

Reply:
good idea